app/src/js/components/home.js

```diff
@@ -193,9 +218,10 @@ class Home extends React.Component {
               <div className="heading__wrapper">
                 <h2 className='heading--medium heading--shared-content--right'>{strings.granules_updated}<span className='num-title'>{numGranules}</span></h2>
               </div>
-
-              <GranulesProgress granules={granuleStatus} />
             </div>
+
+            {this.renderButtonListSection(updated, '')}
```

Comment:
I think we should either pass the header to this function or find a way to not render one if this is blank. Right now we're rendering two `heading__wrapper` divs

Also, unrelated, but I noticed Helmet was being rendered in that `renderButtonListSection` so it's on the page multiple times

If you're already making a change, I would suggest moving that into the main render function so we only have it once.

Reply:
Thanks @dopeters! I just pushed a change that updates `renderButtonListSelection` to accept a complete header element rather than text. It's not the DRYest solution but it might be the clearest. Lemme know if you'd prefer something different.

I also moved `Helmet` to the main render function. It looks like it doesn't totally matter where it is but I put it in the main `page__home` container at the top level.